### PR TITLE
feat(dwellingsList): add DwellingsList component

### PR DESCRIPTION
**Pull request by maximilianoforlenza**
feat(dwellingsList): add dwellingsList component

### CODE REVIEW

File: src/services/survey.js [hunk 1]
```diff
@@ -65,13 +65,18 @@ export default class SurveysService {
...
+    static createHousehold(dwelling) {
+        const newDwelling = dwelling;
+        if (!isEmpty(newDwelling.households)) {
+            const maxOrder = max(map(newDwelling.households, household => household.order));
+            newDwelling.households.push(new Household({order: maxOrder + 1}));
         } else {
-            newSurvey.dwellings[0].households.push(new Household());
+            newDwelling.households.push(new Household());
         }
+        return newDwelling;
```
Comment (navarroaxel) [inline src/services/survey.js]:
```javascript
/**
     * Append a household to the given dwelling
     * @param {Dwelling} dwelling A dwelling to add a household.
     * @returns {Dwelling} The given dwelling including the new household.
     */
    static addHouseholdToDwelling(dwelling) {
        const maxOrder = max(map(dwelling.households, household => household.order)) || 0;
        dwelling.households.push(new Household({order: maxOrder + 1}));
        return dwelling;
    }
```

File: src/components/DwellingsList/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        match: matchParamsIdPropTypes.isRequired,
+        survey: PropTypes.shape({
+            _id: PropTypes.string.isRequired,
+            address: PropTypes.instanceOf(Address).isRequired
+        }),
+        onSelect: PropTypes.func.isRequired
+    };
+
+    static defaultProps = {
+        survey: {}
```
Comment (navarroaxel) [inline src/components/DwellingsList/index.js]: `null`

File: src/components/DwellingResponse/index.js [hunk 3]
```diff
@@ -28,7 +31,6 @@ class DwellingResponse extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            survey: new Survey(),
             submitButtonText: 'Siguiente'
         };
     }
```
Comment (navarroaxel) [inline src/components/DwellingResponse/index.js]:
```javascript
componentDidMount() {
        const {id, order} = this.props.match.params;
        this.props.requestDwelling(id, order);
    }
```

File: src/components/DwellingResponse/index.js [hunk 4]
```diff
@@ -39,25 +41,30 @@ class DwellingResponse extends Component {
...
 
     componentWillReceiveProps(nextProps) {
```
Comment (navarroaxel) [inline src/components/DwellingResponse/index.js]:
```javascript
componentWillReceiveProps(nextProps) {
        if (nextProps.dwelling) {
            this.state.dwelling = nextProps.dwelling;
        }
    }
```

File: src/components/DwellingResponse/index.js [hunk 5]
```diff
@@ -39,25 +41,30 @@ class DwellingResponse extends Component {
...
             this.state.survey = new Survey(survey);
+            if (!dwelling) {
+                this.props.requestDwelling(this.state.survey, this.props.match.params.order);
+            } else {
+                this.state.dwelling = dwelling;
+            }
         }
     }
 
     handleChangeAnswer(answer) {
```
Comment (navarroaxel) [inline src/components/DwellingResponse/index.js]:
```javascript
handleChangeAnswer(answer) {
        this.setState(state => ({
            dwelling: Object.assign(state.dwelling, answer)
        }));
    }
```

File: src/components/DwellingResponse/index.js [hunk 6]
```diff
@@ -39,25 +41,30 @@ class DwellingResponse extends Component {
...
 
     createHousehold() {
-        const {survey} = this.state;
-        if (survey.dwellings[0].dwellingState.response === answers.YES) {
-            this.props.requestCreateHousehold(survey);
+        const {dwelling} = this.state;
+        if (dwelling.dwellingState.response === answers.YES) {
+            this.props.requestCreateHousehold(dwelling);
         }
     }
```
Comment (navarroaxel) [inline src/components/DwellingResponse/index.js]: goToAddressList sacar survey de `props`

File: src/components/DwellingResponse/index.js [hunk 7]
```diff
@@ -67,24 +74,29 @@ class DwellingResponse extends Component {
...
 
     save() {
```
Comment (navarroaxel) [inline src/components/DwellingResponse/index.js]: mover la lógica del save a un sagas/service.

**Comment by maximilianoforlenza** [comment]
listo.

**Pull request merged**